clustering/api/src/main/java/org/wildfly/clustering/group/Group.java

```diff
@@ -105,8 +105,15 @@ default void membershipChanged(Membership previousMembership, Membership members
     Membership getMembership();
 
     /**
-     * Indicates whether this is a local group.  A local group only ever contains a single member.
-     * @return true, if this is a local group, false otherwise.
+     * @deprecated Replaced by {@link #isSingleton()}.
```

Comment:
Generally speaking, I think in most cases the original Javadoc should be kept in place when deprecating -- though in this case (and seems like in all others) the updates are direct replacements, so its fine I think.